index.js

```diff
@@ -70,10 +77,13 @@ function addCommentsToCard(trello, cardId, comments, name, cb){
       log.info('pivotal-to-trello', 'adding comment: %s for %s', comment.text, name);
     }
     var commentPayload = {
-      text: comment.text
+      text:
+        (comment.author?'**Author:** '+comment.author+'\n':'') +
+        (comment.noted_at?'**Date:** '+comment.noted_at+'\n':'') +
+        (comment.text?'\n**Comment:**\n'+comment.text.replace(/^/gm,'> ')+'\n':'')
     };
     tasks.push(function(cb){
-      trello.post('/1/cards/'+cardId+'/comments', commentPayload, cb);
+      trello.post('/1/cards/'+cardId+'/actions/comments', commentPayload, cb);
```

Comment:
It's actually a change in the Trello API – we could make a pull request to the original repo for that.